packages/icons/icons.json

```
@@ -1486,126 +1486,126 @@
     {
         "displayName": "Lightbulb",
         "iconName": "lightbulb",
-        "tags": "idea, glow, lamp",
+        "tags": "idea, glow, lamp, light",
         "group": "miscellaneous",
         "codepoint": 61995
     },
     {
         "displayName": "Glass",
         "iconName": "glass",
-        "tags": "glassware, drink",
+        "tags": "glassware, drink, cheers, cup, wine glass, wine, chalice, goblet",
         "group": "miscellaneous",
         "codepoint": 61905
     },
     {
         "displayName": "Tint",
         "iconName": "tint",
-        "tags": "drop, color, coloration, hue",
+        "tags": "drop, color, coloration, hue, tear, rain, water, rain drop, droplet",
         "group": "media",
         "codepoint": 62197
     },
     {
         "displayName": "Flash",
         "iconName": "flash",
-        "tags": "light, contrast, photograph, picture",
+        "tags": "light, contrast, photograph, picture, sun, star, illuminate, examine, brighten",
         "group": "media",
         "codepoint": 61872
     },
     {
         "displayName": "Font",
         "iconName": "font",
-        "tags": "scale, typography, size",
+        "tags": "scale, typography, size, a, letter, alphabet, letter a, area",
         "group": "editor",
         "codepoint": 61887
     },
     {
         "displayName": "Header",
         "iconName": "header",
-        "tags": "typography, section, layout",
+        "tags": "typography, section, layout, intersections, letter h, track segment, hotel, helicopter, helipad, capital h, headline",
         "group": "editor",
         "codepoint": 61925
     },
     {
         "displayName": "Saved",
         "iconName": "saved",
-        "tags": "document, check mark, tick",
+        "tags": "document, check mark, tick, approval, document ready, checks complete, checklist, approved document, completed document, verified document, confirm, valid, save, upload finished, confirmation, validation, checked, approved, done document, completed, certificate",
         "group": "file",
         "codepoint": 62109
     },
     {
         "displayName": "Floppy disk",
         "iconName": "floppy-disk",
-        "tags": "save",
+        "tags": "save, disk, save as, disk drive, disc drive",
         "group": "interface",
         "codepoint": 61873
     },
     {
         "displayName": "Book",
         "iconName": "book",
-        "tags": "pages, album, brochure, manual",
+        "tags": "pages, album, brochure, manual, bookmark, notebook, journal, documentation, reading, instructions, file, cathedral",
```

```suggestion
        "tags": "pages, album, brochure, manual, bookmark, notebook, journal, documentation, reading, instructions, file",
```

I'm not sure how cathedral relates...?

---

i guess it kind of looks like one lol